packages/meditrak-server/src/dataAccessors/index.js

```diff
@@ -4,10 +4,6 @@
  */
```

Comment:
Surprised there are still so many left! Where is e.g. `findSurveyScreenComponentsInSurvey` used?